CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## [Unreleased]
 
+## RELEASE 0.4.3 - 2016-11-17
```

Comment:
Hey @frisk!

Thank you for the complementary PR, that's great to have guys like you to make Forest better!

We'll probably accept your PR quickly but we might not release the feature today.
What we usually do is that we had the release title once it is effectively released.
Can you keep your features under the [Unreleased] title?

Thank you!